spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -479,7 +513,7 @@ def __init__(self, parent=None):
         self.__find_first_pos = None
         self.__find_flags = None
 
-        self.language = None
+        self.lsp_language = None

```

Comment:
i really don't see the need to change this variable name to `lsp_language`. Since it's used in a lot of places and it's quite critical for the LSP to work correclty, could it be reverted to simply `language`?